src/transport/tests/TestSecureSessionMgr.cpp

```diff
@@ -160,6 +160,7 @@ void CheckMessageTest(nlTestSuite * inSuite, void * inContext)
 
     TransportMgr<LoopbackTransport> transportMgr;
     SecureSessionMgr secureSessionMgr;
+    PayloadHeader payloadHeader;
```

Comment:
We should set a useful protocol/message-id on this payload header.

Reply:
Acked